ros-ardent/sros2/sros2-0.4.1.ebuild

```diff
@@ -8,7 +8,7 @@ inherit ament-python
 
 DESCRIPTION="Command line tools for managing SROS2 keys"
 HOMEPAGE="https://wiki.ros.org"
-SRC_URI="https://github.com/ros2-gbp/${PN}-release/archive/release/ardent/${PN}/0.4.0-0.tar.gz -> ${PN}-ardent-release-${PV}.tar.gz"
+SRC_URI="https://github.com/ros2-gbp/${PN}-release/archive/release/ardent/${PN}/0.4.1-0.tar.gz -> ${PN}-ardent-release-${PV}.tar.gz"
```

Comment:
@mikaelarguedas :tada: :taco: :tada:

Reply:
(first ROS 2 update for Gentoo)

Reply:
You may want to hold for a day as we're about to sync: https://github.com/ros2/ros2/issues/472

Reply:
Nevermind this is already merged ^^

Reply:
Ya, I'll maybe start following with Debian syncs one day...